.travis.yml

```diff
@@ -3,6 +3,6 @@ before_install:
   - sed -i.bak -e 's|https://nexus.codehaus.org/snapshots/|https://oss.sonatype.org/content/repositories/codehaus-snapshots/|g' ~/.m2/settings.xml
 script:
         mvn clean test &&
-        mvn clean test -Pall
+        mvn clean test -Pall -DfailIfNoTests=false
```

Comment:
`Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (wf10.1) on project arquillian-container-chameleon-parent: No tests were executed!  (Set -DfailIfNoTests=false to ignore this error.) `

Wouldn't it be better to fix it